Corpora/csip/metadata/amdSec/csip32

andersbonielsen opened the pull request:
Ready for review. @carlwilson or @ltaimre can look at the structMap part and @PhillipTommerholt for the amdSec part.

andersbonielsen committed:
Created test case description and corpus

Corpus missing valid premis file and premis schema. Not necessary for testing CSIP32, but should be there.

andersbonielsen committed:
Added div elements in structMap for MUST folders

PhillipTommerholt commented on corpora/csip/metadata/amdsec/CSIP32/test-case.xml:
```diff
@@ -1,23 +1,88 @@
+      </error>
+      <corpusPackages>
+        <!-- Details of corpus packages designed to test this validation rule. -->
+        <package name="IP_18000_CSIP32_1" isValid="FALSE">
```
isValid should be TRUE according to the new definition in the schema

PhillipTommerholt commented on corpora/csip/metadata/amdsec/CSIP32/test-case.xml:
```diff
@@ -1,23 +1,88 @@
+      </error>
+      <corpusPackages>
+        <!-- Details of corpus packages designed to test this validation rule. -->
+        <package name="IP_18000_CSIP32_2" isValid="FALSE">
```
isValid should be TRUE according to the new definition in the schema for warnings

PhillipTommerholt commented on corpora/csip/metadata/amdsec/CSIP32/IP_18000_CSIP32_3/METS.xml:
```diff
@@ -0,0 +1,102 @@
+  xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance schemas/XMLSchema.xsd http://www.loc.gov/METS/ schemas/mets.xsd 
+    http://www.w3.org/1999/xlink schemas/xlink.xsd 
+    https://DILCIS.eu/XML/METS/CSIPExtensionMETS schemas/CSIPExtensionMETS.xsd"
+  OBJID="IP_18000_CSIP17_3" TYPE="Database" csip:CONTENTINFORMATIONTYPE="SIARD2"
```
wrong OBJID

PhillipTommerholt reviewed (changes requested):
Made some few comments. Also: rename test-case.xml to testCase.xml

PhillipTommerholt committed:
Updated testCase.xml and corpus packages

Updated testCase.xml and corpus packages - but I guess there are some errors in the packages

carlwilson committed:
Merge branch 'integration' into corpora/csip/metadata/amdSec/CSIP32

carlwilson committed:
Merge branch 'integration' into corpora/csip/metadata/amdSec/CSIP32
